src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_AutoSuggestBox.cs

```diff
@@ -56,6 +56,22 @@ public async Task When_Text_Changed_And_Focused_Should_Open_Suggestion_List()
 			SUT.IsSuggestionListOpen.Should().BeTrue();
 		}
 #endif
+		[TestMethod]
+		public async Task When_Text_IsNull()
+		{
+			var sut = new AutoSuggestBox();
+
+			WindowHelper.WindowContent = sut;
+
+			await WindowHelper.WaitForIdle();
+
+			sut.Text = null;
+
+			var textBox = (TextBox)sut.GetTemplateChild<TextBox>("TextBox");
```

Comment:
This is an Uno-specific extensions, so this makes the UWP build fail:
```
src\Uno.UI.RuntimeTests\Tests\Windows_UI_Xaml_Controls\Given_AutoSuggestBox.cs(70,31): Error CS0308: The non-generic method 'Control.GetTemplateChild(string)' cannot be used with type arguments
```

```suggestion
			var textBox = (TextBox)sut.GetTemplateChild("TextBox") as TextBox;
```